### PR TITLE
Doin' Stuff v2 (Code improvements)

### PR DESCRIPTION
**Pull request by Sigmanificient**
## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [ ] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

File: EpikCord/status_code.py [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+        raise ValueError(
+            f"HTTP Status Code `{value}` is undocumented.\n"
+            "Please create an issue at:"
+            " https://github.com/EpikCord/EpikCord.py/issues"
+            " with the following traceback."
+        )
+
+
+class JsonErrorCodes(StatusCode):
```
Comment (TheUntraceable) [inline EpikCord/status_code.py]: This isn't needed I don't think.

**Review by TheUntraceable — CHANGES_REQUESTED** [review]
If you could move all these into `utils.types`, that'd be great, thanks.

File: EpikCord/status_code.py (hunk 1 above)
Comment (TheUntraceable) [reply EpikCord/status_code.py]: Change of mind.

File: EpikCord/client/http.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
     json_serialize,
     log_request,
 )
+from ..utils.enums import HTTPCodes
```
Comment (TheUntraceable) [inline EpikCord/client/http.py]: You don't need the `.enums` part, just import from `utils`.

**Review by TheUntraceable — CHANGES_REQUESTED** [review]

File: EpikCord/client/http.py (hunk 2 above)
Comment (Sigmanificient) [reply EpikCord/client/http.py]: > You don't need the `.enums` part, just import from `utils`.

```suggestion
from ..utils import HTTPCodes
```

Comment (Sigmanificient) [reply EpikCord/client/http.py]: image

**Review by TheUntraceable — APPROVED** [review]

**Pull request merged**